### PR TITLE
Bugfix - Revert bad merge

### PR DESCRIPTION
**Pull request by nickzuber**
Reverts nickzuber-alt-but-cooler/test#3

### CODE REVIEW

**Review by nickzuber-alt-but-even-cooler — APPROVED** [review]

**Review by nickzuber — COMMENTED** [review]
my own comment

**Review by nickzuber — COMMENTED** [review]
a review.

**Review by nickzuber — COMMENTED** [review]
another.

**Review by nickzuber — COMMENTED** [review]
one mroe.

**Review by nickzuber — COMMENTED** [review]
ghdf

**Review by nickzuber — COMMENTED** [review]
ggg

**Review by nickzuber — COMMENTED** [review]
czccc

**Review by nickzuber — COMMENTED** [review]
ok got it

**Review by nickzuber — COMMENTED** [review]
@nickzuber-alt-but-cooler hey

**Comment by nickzuber** [comment]
fsdf

**Comment by nickzuber** [comment]
df

**Comment by nickzuber** [comment]
sfd

**Comment by nickzuber** [comment]
gdfg

**Comment by nickzuber** [comment]
dsdf

**Comment by nickzuber-alt-but-even-cooler** [comment]
hey

File: utils.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 export function test () {
-  return 123
+  return null
```
Comment (nickzuber-alt-but-even-cooler) [inline utils.ts:2]: Maybe we can do a forward fix instead?

**Review by nickzuber-alt-but-even-cooler — APPROVED** [review]

File: utils.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
 export function test () {
-  return 123
```
Comment (nickzuber-alt-but-even-cooler) [inline utils.ts:2]: Wait I like numbers

**Review by nickzuber-alt-but-even-cooler — CHANGES_REQUESTED** [review]

**Comment by nickzuber-alt-but-even-cooler** [comment]
:)

**Comment by nickzuber-alt-but-even-cooler** [comment]
@nickzuber yo

**Comment by nickzuber-alt-but-even-cooler** [comment]
@nickzuber hi

**Pull request merged**